x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -508,32 +506,52 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     type: string | string[],
     options: SavedObjectsOpenPointInTimeOptions
   ) {
-    try {
-      const args = { type, options };
-      await this.legacyEnsureAuthorized(type, 'open_point_in_time', options?.namespace, {
+    const args = { type, options };
+    const { status, typeMap } = await this.legacyEnsureAuthorized(
+      type,
+      'open_point_in_time',
+      options?.namespaces,
+      {
         args,
         // Partial authorization is acceptable in this case because this method is only designed
         // to be used with `find`, which already allows for partial authorization.
         requireFullAuthorization: false,
-      });
-    } catch (error) {
+      }
+    );
+
+    if (status === 'unauthorized') {
       this.auditLogger.log(
         savedObjectEvent({
           action: SavedObjectAction.OPEN_POINT_IN_TIME,
-          error,
+          error: new Error(status),
         })
       );
-      throw error;
+      throw SavedObjectsErrorHelpers.decorateForbiddenError(new Error(status));
```

Comment:
In the same scenario, the `find` wrapper returns `SavedObjectsUtils.createEmptyFindResponse`, however this is not possible for `openPointInTimeForType`, as the expected return type is a `SavedObjectsOpenPointInTimeResponse`. I'm throwing a forbidden error in that case, but really not sure this is the correct error type?

Reply:
Yeah, the forbidden error seems like the best approach here.

Reply:
>  if you are only authorized to access a type in space A, and you attempt to open a PIT for that type only in space B, you'll run into a forbidden error. But that seems acceptable.

I would even think that this is expected and not just acceptable?

Reply:
Hum, actually we can't do that, as it would need to be done in `createPointInTimeFinder`, so it doesn't solve the issue when calling `openPointInTimeForType` directly. So throwing here seems the only viable option anyway